tools/reviewer/job/sync/DiffConverter.java

```diff
@@ -115,37 +117,44 @@ private String getPullRequestState(Diff.Status status) {
     }
   }
 
-  private int getReviewCommentPositionFromGitHub(Thread thread, GithubPr githubPr) {
+  private int getReviewCommentPositionFromGitHub(Thread thread, String repoOwner, String repoName) {
     LineNumberConverter.Side commentSide =
         thread.getFile().getCommitId().equals(thread.getCommitId())
             ? LineNumberConverter.Side.RIGHT
             : LineNumberConverter.Side.LEFT;
-    String patch = "";
+
+    List<String> githubFilePatches = new ArrayList<>();
     try {
-      patch =
+      githubFilePatches =
           githubClient
               .getCommit(
                   CommitRequest.newBuilder()
-                      .setOwner(githubPr.getOwner())
-                      .setRepo(githubPr.getRepo())
+                      .setOwner(repoOwner)
+                      .setRepo(repoName)
                       .setSha(thread.getFile().getCommitId())
                       .build())
               .getCommit()
               .getFilesList()
               .stream()
-              .filter(file -> thread.getFile().getFilename().equals(file.getFilename()))
-              .collect(Collectors.toList())
-              .get(0)
-              .getPatch();
+              .filter(githubFile -> thread.getFile().getFilename().equals(githubFile.getFilename()))
+              .map(CommitInfo.File::getPatch)
+              .collect(Collectors.toList());
     } catch (IOException e) {
       e.printStackTrace();
     }
 
+    if (githubFilePatches.size() != 1) {
+      throw new RuntimeException(
+          "Can't find the commit in GitHub for the file: " + thread.getFile().getFilename());
```

Comment:
You actually did find the commit (`getCommit()`) returned an object, so I suggest:
s/Can't find the commit in GitHub for the file/Expecting only 1 patch in response for file: